Changefeed/ChangefeedSample/ChangefeedSample.cs

```diff
@@ -0,0 +1,74 @@
+         * Before starting the program, delete the cursor from previous runs from the storage account, if it exists
+         */
+        [FunctionName("Function1")]
+        public static async void Run([TimerTrigger("0 */1 * * * *")]TimerInfo myTimer, ILogger log)
+        {
+            string connectionString = "CONNECTION_STRING";
+            string containerName = "example";
+
+            //Predicate to filter events from Changefeed
+            Predicate<BlobChangeFeedEvent> containerCheck = (BlobChangeFeedEvent changeFeedEvent) => { return changeFeedEvent.Subject.Contains("/containers/" + containerName + "/"); };
+                        
+            BlobServiceClient blobServiceClient = new BlobServiceClient(connectionString);
+            BlobChangeFeedClient changeFeedClient = blobServiceClient.GetChangeFeedClient();
+            BlobContainerClient containerClient = blobServiceClient.GetBlobContainerClient("storingcursorcontainer");
+            BlobClient blobClient = new BlobClient(connectionString, "cursorcontainer","storingcursorblob");                      
+                      
+            string continuationToken = null;
+
+            if(blobClient.Exists())
+            {
+                //If the continuationToken exists in blob, download and use it
+                var stream = new MemoryStream();
+                blobClient.DownloadTo(stream);
+                continuationToken = Encoding.UTF8.GetString(stream.ToArray());
+            }
+            
+            else
+            {
+                //If the continuationToken does not exist in the blob, get the continuationToken from the first item
+                await foreach (Page<BlobChangeFeedEvent> page in changeFeedClient.GetChangesAsync().AsPages(pageSizeHint: 1))
+                {
+                    foreach (BlobChangeFeedEvent changeFeedEvent in page.Values)
+                    {
+                        log.LogInformation($"event: {changeFeedEvent.EventType} at {changeFeedEvent.Subject.Replace("/blobServices/default", "")} on {changeFeedEvent.EventTime}");
+                    }
+                    continuationToken = page.ContinuationToken;
+                    break;
+                }
+            }
+            
+            //Loop through the events in Changefeed with the continuationToken
+            await foreach (Page<BlobChangeFeedEvent> page in changeFeedClient.GetChangesAsync(continuationToken: continuationToken).AsPages(pageSizeHint: 1))
+            {
+                foreach (BlobChangeFeedEvent changeFeedEvent in page.Values)
+                {
+                    if (containerCheck.Invoke(changeFeedEvent))
+                    {
+                        log.LogInformation($"event: {changeFeedEvent.EventType} at {changeFeedEvent.Subject.Replace("/blobServices/default","")} on {changeFeedEvent.EventTime}");
+                    }                    
+                }
+                continuationToken = page.ContinuationToken;
```

Comment:
So the continuation token get changed every loop, or does this loop only run once cause you only expect 1 page

Reply:
currently, the continuation token gets changed every loop